ingest/scripts/heuristic_group_segments.py

```diff
@@ -211,6 +211,11 @@ def main(
                 if segment in group
             ]
         )
+        if not joint_key:
```

Comment:
this is only if nextclade sort finds a match which we do not have in our "known matches"